plugins/scaffolder/src/components/MultistepJsonForm/schema.ts

```diff
@@ -120,7 +120,8 @@ export function transformSchemaToProps(
     )?.component;
 
     if (Layout) {
-      uiSchema['ui:ObjectFieldTemplate'] = Layout;
+      uiSchema['ui:ObjectFieldTemplate'] =
```

Comment:
Hmm this cast is unfortunate. I guess it's because it's using the `uiSchema` type from the later versions of `@rjsf/*` right?

Reply:
@benjdlambert pretty much, in the `@rjsf/core-v5` schema, the `ObjectFieldTemplate` uses `ComponentType`

```ts
ObjectFieldTemplate: React.ComponentType<ObjectFieldTemplateProps<T, S, F>>;
```

But in `@rjsf/core-3.2.0`, it uses `StatelessComponent`

```ts
'ui:ObjectFieldTemplate'?: React.StatelessComponent<ObjectFieldTemplateProps>;
```

`ComponentType` is a union of `ComponentClass` and `StatelessComponent` so typescript will not do the conversion because the 3.2.0 type is only for `StatelessComponent` and not `ClassComponent`.